src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/__init__.py

```diff
@@ -179,11 +180,28 @@ def _handle_env_configuration():
         # no env exists so create first environment
         _create_or_update_env('default')
 
+def _handle_telemetry_configuration():
+    allow_telemetry = prompt_y_n(MSG_PROMPT_TELEMETRY, default='n')
+
+    # save the global config
+    global_config = configparser.SafeConfigParser()
+    global_config.read(GLOBAL_CONFIG_PATH)
+    try:
+        global_config.add_section('telemetry')
+    except configparser.DuplicateSectionError:
+        pass
+    global_config.set('telemetry', 'collect', 'y' if allow_telemetry else 'n')
```

Comment:
Instead of setting to 'y' or 'n', you can pass in True/False to when we get the config value, we can use getboolean()